duckduckgo/macos-browser
# bookmark icon and passwords popover quick fix

Task/Issue URL:
* https://app.asana.com/0/inbox/392891325557408/1201065654004996/1201066012835602
* https://app.asana.com/0/inbox/392891325557408/1201065654004988/1201066012835602

Tech Design URL:
CC:

**Description**:

Bookmark icon appears in the address bar only on hover and uses the system accent color.

**Steps to test this PR**:
1. Check that the icon appears and is hidden appropriate (e.g. when the field is being edited, etc)
1. Check that the passwords and logins popover disappears when you click the button in the navigation bar while it is showing (like bookmarks)

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
